sections/featured-product-slider.liquid

```diff
@@ -125,7 +105,8 @@
     {
       "type": "text",
       "id": "title",
-      "label": "Title"
+      "label": "Title",
+      "title": "عرض"
```

Comment:
add default instead of "title"
also provide a default color for the Title color